aws/resource_aws_athena_named_query.go

```diff
@@ -29,6 +29,11 @@ func resourceAwsAthenaNamedQuery() *schema.Resource {
 				Required: true,
 				ForceNew: true,
 			},
+			"workgroup": {
+				Type:     schema.TypeString,
+				Optional: true,
+				ForceNew: true,
```

Comment:
When running the updated acceptance testing (thank you for splitting them out, it discovered this), it appears that Athena uses a default workgroup named `primary` that all named queries get assigned to if they are not customized. This is not called out in the [Athena API Reference](https://docs.aws.amazon.com/athena/latest/APIReference/API_CreateNamedQuery.html#athena-CreateNamedQuery-request-WorkGroup) but it does have a small mention in the [Athena User Guide](https://docs.aws.amazon.com/athena/latest/ug/user-created-workgroups.html).

Here's the test failure I was seeing:

```
--- FAIL: TestAccAWSAthenaNamedQuery_basic (39.62s)
    testing.go:568: Step 0 error: After applying this step, the plan was not empty:

        DIFF:

        DESTROY/CREATE: aws_athena_named_query.foo
          database:    "7syob" => "7syob"
          description: "tf test" => "tf test"
          id:          "e27863d7-77e5-4b70-9662-69cf4346bec1" => "<computed>"
          name:        "tf-athena-named-query-7syob" => "tf-athena-named-query-7syob"
          query:       "SELECT * FROM 7syob limit 10;" => "SELECT * FROM 7syob limit 10;"
          workgroup:   "primary" => "" (forces new resource)
```

To account for this in the Terraform resource, we need to set a matching default:

```suggestion
				ForceNew: true,
				Default:  "primary",
```

Reply:
ah, I thought this was due to me clobbering the first resource with my second test. :-) I've slightly altered the test cases to avoid this, taking an example from another resource and no longer run into this issue.

Still, setting `Default: "primary"` seems sane. Will test again and push in a bit.

Reply:
That's done it. Though I have created a completely new test case (following another resources pattern) rather than the additional step I'd used previously. Sorry, first time for me with Go and unfamiliar with the tests. Happy to revert back to steps if you'd prefer.

Reply:
Splitting it out into its own test is a great pattern! 😄 This allows each test scenario to run separately in parallel and allows the test naming to provide a hint about what piece of functionality is having trouble when something goes awry. This PR looks really good, just running the acceptance testing again before merging. 👍